### DISCUSSION TITLE
Write ctf file with grain mean orientation

### DISCUSSION THREAD

**Discussion by SunilAnandatheertha** [mtex-toolbox/mtex#2219] [Ask Anything]
Hi all, could we write a ctf file with all pixels mapped with mean orientations of the grains that they belong to?
Thank you.

**Comment by miamaric12** [comment]
This would be very helpful, i am also struggling to do this

**Comment by ralfHielscher** [comment]
I think a problem is not so much to write this into the .ctf file but rather wich program can read it afterwards. The ctf format is very little standardized. Do you know in which format your program expects this information.

Ralf.

**Comment by SunilAnandatheertha** [comment]
Something of this sort would be useful:

```
Channel Text File
Prj	PROJECT_NAME
Author	Dr. Sunil Anandatheertha
JobMode	Grid
XCells	51
YCells	51
XStep	1.0
YStep	1.0
AcqE1	0
AcqE2	0
AcqE3	0
Euler angles refer to Sample Coordinate system (CS0)!	Mag	0.0000	Coverage	0	Device	0	KV	0.0000	TiltAngle	0.0000	TiltAxis	0	DetectorOrientationE1	0.0000	DetectorOrientationE2	0.0000	DetectorOrientationE3	0.0000	WorkingDistance	0.0000	InsertionDistance	0.0000
Phases	1
3.614;3.614;3.614	90.000;90.000;90.000	Copper	11	0			Created from NAME
Phase	X	Y	Bands	Euler1	Euler2	Euler3	Error	MAD	BC	BS
1.0	0.000	0.000	8	228.42846	156.38913	336.14185	0	0.36211	204.0	170.0
```

In the above, I have made the Bands of all pixels same, randomized the Error, MAD, BC and BC as these are columns which will not be used by me.  Its only the Phase, X, Y and Euler angles, which are useful for the case I am working with.
